bin/ham-gdb

```diff
@@ -5,22 +5,13 @@ if [[ -z "$HAM_HOME" ]]; then echo "E/HAM_HOME not set !"; exit 1; fi
 
 case $HAM_OS in
     NT*)
-        export MSVC_IDE_DIR="`unxpath "$PROGRAMFILES\\Microsoft Visual Studio 12.0\\Common7\\IDE"`"
-        export RUN_DEBUGGER="${MSVC_IDE_DIR}/devenv.exe"
         if [ ! -f "$RUN_DEBUGGER" ]; then
```

Comment:
It looks odd, its to handle the case where its set globally and somehow you're not running from a ham-shell?

Reply:
Mmmh, I assume that a toolset might have been setup and RUN_DEBUGGER could have been exported already, so I just check before searching for the IDE path needlessly.

But to be frank I don't know when exactly these are called, and who calls them so I'm just trying to cover my bases from what I know ;-)

Reply:
`ham-gdb` is called directly by a user in the terminal or by ham when using the `Debug_` targets.

> Mmmh, I assume that a toolset might have been setup and RUN_DEBUGGER
Yes, that could be the case. I guess my point (which I failed to articulate ^^) was that if that's the intent then it needs a comment since its not obvious.

Reply:
Added the comment